src/www/ui/template/ui-report-conf.html.twig

```diff
@@ -121,26 +121,29 @@
           </td>
           <td align="left">
             <label><input type="checkbox" name="noDependency" {{ noDependency }} />{{ "no dependencies found, neither in source code nor in binaries"|trans }}</label><br />
-            <label><input type="checkbox" name="dependencySource" {{ dependencySource }} />{{ "dependencies found in source code (see obligations)"|trans }}</label><br />
-            <label><input type="checkbox" name="dependencyBinary" {{ dependencyBinary }} />{{ "dependencies found in binaries (see obligations)"|trans }}</label>
+            <label><input type="checkbox" onclick="toggleTextArea1();" id="dependencySource" name="dependencySource" {{ dependencySource }} />{{ "dependencies found in source code (see obligations)"|trans }}</label><br />
```

Comment:
Please update the function name from `toggleTextArea1` to something more meaningful.